Deploy-CAM.ps1

```diff
@@ -1008,7 +1028,7 @@ function New-CAMAppSP() {
 
     # Get tenant ID for this subscription
     $subForTenantID = Get-AzureRmContext
-    $tenantID = $subForTenantID.Tenant.Id
+    $tenantID = $subForTenantID.Tenant.TenantId
```

Comment:
Were you having an error here? I can't remember which version is 'the future'...

Reply:
I was when I was stepping through the code, tenenatID was always null

Reply:
Hmmm- which version? I am seeing both populated here. I think I'm on 5.0.1

Reply:
@roy-satadru - do you recall which one was 'gone' in 5.0.0 - Tenant.Id or Tenant.TenantId ?

Reply:
Sorry, I had missed this. Yes, I think Tenant.TenantId was removed from 5.0.0. So was Subscription.SubscriptionId and instead they only had the attributes Id. In 5.0.1 they added the longer attributes back.

Reply:
Okay, so we're guessing which one will cause the least pain going forward. Since we don't know I guess I'm fine with the change :)